components/icon/index.en-US.md

```diff
@@ -26,6 +26,7 @@ ReactDOM.render(<IconDisplay />, mountNode);
 | style | Style properties of icon, like `fontSize` and `color` | CSSProperties | - |
 | theme | Theme of the ant design icon  | 'filled' \| 'outlined' \| 'twoTone' | 'outlined' |
 | spin | Rotate icon with animation | boolean | false |
+| rotate | rateto degress (added in 3.13.0, not working in IE9) | number | - |
```

Comment:
Should this be ``Rotate to degrees`` or ``Rotate degrees``

Reply:
Fixed in 0b3d7d62fa